packages/system/capi-operator/Makefile

```diff
@@ -1,11 +1,21 @@
 NAME=capi-operator
 NAMESPACE=cozy-cluster-api
+TYPE=system
 
+include ../../../scripts/common-envs.mk
 include ../../../scripts/package-system.mk
 
-update:
+update: update-charts update-dockerfiles
+
+update-charts:
 	rm -rf charts
 	helm repo add capi-operator https://kubernetes-sigs.github.io/cluster-api-operator
 	helm repo update capi-operator
 	helm pull capi-operator/cluster-api-operator --untar --untardir charts
 	rm -rf charts/cluster-api-operator/charts
+
+update-dockerfiles:
+	../../../scripts/update-dockerfiles.sh with_helm
```

Comment:
Personally I don't have anything against of helper scripts to combine common logic, however they should be located in `/hack` directory.